CMakeLists.txt

```diff
@@ -124,7 +124,8 @@ endif()
 # ==============================================================================
 option(CUDAQ_BUILD_TESTS "Build cudaq tests" ON)
 option(CUDAQ_ENABLE_RPC_LOGGING "Enable verbose printout for client/server qpud connection." OFF)
-option(CUDAQ_TEST_MOCK_SERVERS "Enable Remote QPU Tests via Mock Servers." OFF)
+option(CUDAQ_TEST_MOCK_SERVERS "Enable Remote QPU Tests via Mock Servers." ON)
```

Comment:
Is this change of the default value for `CUDAQ_TEST_MOCK_SERVERS` intentional?

Reply:
I don't think we should change the default here.

Reply:
No, this wasn't intentional. Will revert this.